src/palace/manager/sqlalchemy/model/classification.py

```diff
@@ -355,7 +355,7 @@ class Classification(Base):
     # trust the distributor to have accurate classifications), we
     # should give it this weight. This lets us keep the weights
     # consistent across distributors.
-    TRUSTED_DISTRIBUTOR_WEIGHT = 100.0
+    TRUSTED_DISTRIBUTOR_WEIGHT = 100
```

Comment:
We had this set as a float, but its stored in the DB as an `int`, so we update it to match here.